integration/tests/params.go

```diff
@@ -217,3 +226,43 @@ func NewParams(clusterNamePrefix string) *Params {
 	params.GenerateCommands()
 	return &params
 }
+
+// attemptSettingUserID will attempt to fetch the first 2 characters of the userID running the
+// integration tests to be used as a prefix for cluster names. It fails silently if an error occurs
+func (p *Params) attemptSettingUserID() {
+	clusterProvider, err := eks.New(&api.ProviderConfig{Region: p.Region}, &api.ClusterConfig{
+		TypeMeta: api.ClusterConfigTypeMeta(),
+		Metadata: &api.ClusterMeta{
+			Version: p.Version,
+			Name:    p.ClusterName,
+			Region:  p.Region,
+		},
+	})
+
+	msg := "Warning: failed to get UserID, proceeding without configuring cluster name prefix. Error: %w"
+	if err != nil {
+		fmt.Fprintf(GinkgoWriter, msg, err)
+		return
+	}
+
+	out, err := clusterProvider.Provider.STS().GetCallerIdentity(&sts.GetCallerIdentityInput{})
+	if err != nil {
+		fmt.Fprintf(GinkgoWriter, msg, err)
+		return
+	}
+
+	if out.UserId == nil {
+		fmt.Fprintf(GinkgoWriter, msg, fmt.Errorf("UserId value not set in AWS response"))
+	}
+
+	//example value: ABCD1234:my-name@email-provider
+	parts := strings.Split(*out.UserId, ":")
+	if len(parts) == 2 {
+		parts = strings.Split(parts[1], "@")
+		if len(parts) == 2 {
+			p.userID = parts[0][0:2]
```

Comment:
can we use the first 4 chars? it'll make it easier for us to identify who's cluster it is :)

Reply:
sure, bit concerned that the cluster names are getting even longer but I doubt 2 more will do any harm :smile: